practice/4_python_part_3/task_5.py

```diff
@@ -6,10 +6,21 @@
      200, 'response data'
 """
 from typing import Tuple
+import urllib.request
+import ssl
 
+ssl._create_default_https_context = ssl._create_stdlib_context
```

Comment:
Why do you need it?

Reply:
Without it get an error: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000)>
(0, '<urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000)>')